src/components/spatial/tiling-utils.js

```diff
@@ -42,9 +44,10 @@ export function getTileIndices(viewport, maxZoom, minZoom, tileSize) {
       |(roundedMinX)             |(roundedMaxX)
    */
   const roundedMinX = Math.max(0, Math.floor(minX));
-  const roundedMaxX = Math.max(0, Math.ceil(maxX));
+  const roundedMaxX = Math.min(maxXTilePossible, Math.max(0, Math.ceil(maxX)));
   const roundedMinY = Math.max(0, Math.floor(minY));
-  const roundedMaxY = Math.max(0, Math.ceil(maxY));
+  const roundedMaxY = Math.min(maxYTilePossible, Math.max(0, Math.ceil(maxY)));
+
```

Comment:
Have you been working with anything outside the positive/positive quadrant? I don't think we should block on that, if you don't have any examples at hand, but file an issue to check on that case?

Reply:
I suppose we are not but I don't think that we want to return anything that is negative since then we'll make get requests against it.  I assume the projection matrix gives the ability to return negative coordinates. This is just an off-the-cuff "why" so I'll look at it more closely for sure.